src/test/resources/.gitignore

```diff
@@ -0,0 +1,3 @@
+# We blanket-ignore .bam files in the top-level of the repo, but want to undo that here.
+!*.bam
```

Comment:
Yeah originally we had a prohibition on adding binary files, but we've relaxed that over time. Thanks for patching this!